packages/admin/admin/src/form/FinalFormFileUpload.tsx

```diff
@@ -0,0 +1,222 @@
+        onDrop,
+        accept,
+        disabled,
+        multiple: multipleFiles,
+        maxSize: maxFileSizeInBytes,
+    });
+
+    // list of the accepted files
+    const files =
+        fieldValue.length > 0 &&
+        fieldValue.map((file) => (
+            <div key={file.name} className={classes.fileListItem}>
+                {file.name.length < 20 ? file.name : `${file.name.substring(0, 20)}...`}
+                <div>
+                    <Chip label={<PrettyBytes value={file.size} />} />
+                    <IconButton onClick={removeFile(file)}>
+                        <Delete />
+                    </IconButton>
+                </div>
+            </div>
+        ));
+
+    const rejectedFiles =
+        fileRejections.length > 0 &&
+        fileRejections.map((rejectedFile) => (
+            <div key={rejectedFile.file.name} className={classes.rejectedFileListItem}>
+                {rejectedFile.file.name.length < 20 ? rejectedFile.file.name : `${rejectedFile.file.name.substring(0, 20)}...`}
+                <IconButton color="error">
+                    <Info />
+                </IconButton>
+            </div>
+        ));
+
+    return (
+        <div className={classes.root}>
+            <div {...getRootProps()} className={classes.root}>
+                <input {...getInputProps()} />
+                {dropzoneVariant !== "buttonOnly" && (
+                    <div className={clsx(classes.droppableArea, disabled && classes.disabled, isDragReject && classes.error)}>
+                        {isDragReject && (
+                            <div className={classes.droppableAreaError}>
+                                <Info />
+                            </div>
+                        )}
+                        <div className={classes.droppableAreaCaption}>
+                            <FormattedMessage id="comet.finalformfileupload.dropfiles" defaultMessage="Drop files here to upload" />
+                        </div>
+                    </div>
+                )}
+                {dropzoneVariant !== "dropzoneOnly" && (
+                    <Button disabled={disabled} variant="contained" color="primary" startIcon={<Select />}>
+                        <FormattedMessage id="comet.finalformfileupload.selectfile" defaultMessage="Select file" />
+                    </Button>
+                )}
+            </div>
+            {fieldValue.length > 0 && <div className={classes.fileList}>{files}</div>}
+            {fileRejections.length > 0 && <div className={classes.fileList}>{rejectedFiles}</div>}
+            {(fileRejections.length > 0 || isDragReject) && (
+                <div className={classes.errorMessage}>
+                    <Info color="error" />
+                    <FormattedMessage id="comet.finalformfileupload.errors.unknownError" defaultMessage="Something went wrong." />
+                </div>
+            )}
```

Comment:
The max file size should still be shown down here, see screen design:

<img width="473" alt="Bildschirmfoto 2024-01-09 um 10 04 06" src="https://github.com/vivid-planet/comet/assets/13380047/d979d64d-83cc-4de1-aff0-f728fbd8e970">

<img width="470" alt="Bildschirmfoto 2024-01-09 um 10 03 31" src="https://github.com/vivid-planet/comet/assets/13380047/725d0662-0472-4256-80c9-82f1b805feb8">